readthedocs/templates/socialaccount/snippets/provider_list.html

```diff
@@ -7,8 +7,9 @@
   {% comment %}
     - OpenID is not implemented.
     - SAML is handled in another view, we don't want to list all SAML integrations here.
+    - GitHub App is not exposed to users yet.
   {% endcomment %}
-  {% if provider.id != 'saml' %}
+  {% if provider.id != 'saml' and provider.id != 'githubapp' %}
```

Comment:
Instead of skipping it from here, why don't we show the button only to staff users?

Reply:
If you are not logged in, there is no way to tell if you are staff user or not.

Reply:
🙃

Reply:
I need vacations 😄